.gitignore

```diff
@@ -78,3 +78,7 @@
 /libbulletjme.so
 /liblwjgl64.so
 /copyToRaspi.sh
+/lastRestart.py
```

Comment:
are these ignores required?  We should avoid ignores on things in the actual src/ folder to help keep our working copies cleaner..

Reply:
I agree .. the .gitignore deserves cleanup..
lastRestart.py probably deserves to be there - now that restarting mrl is a valid possibility, this is the file which attempt to preserve state between restarts ..
.factorypath is and vscode IDE file ... so probably shoudl be in there too ..
the other two 
certs should be removed ...
and the other one I see should probably be under /data so I'll fix